Task1/Node/fs-demo.js

```diff
@@ -0,0 +1,45 @@
+const ghash = require('./hash-generation.js');
+
+const fs = require("fs");
+const path = fs.readdirSync("./utils");
```

Comment:
instead of assuming the directory to be utils, take it from `process.argv`.

Reply:
modified the code as: 
const [, , directoryname] = process.argv;
var stats = fs.statSync(path.join('./',directoryname));

made use of process.argv.